anesthetic/samples.py

```diff
@@ -289,8 +174,8 @@ def plot_2d(self, axes, *args, **kwargs):
             this is used for creating the plot. Otherwise a new set of axes are
             created using the list or lists of strings.
 
-        types: dict, optional
-            What type (or types) of plots to produce. Takes the keys 'diagonal'
+        kind: dict, optional
+            What kinds of plots to produce. Takes the keys 'diagonal'
             for the 1D plots and 'lower' and 'upper' for the 2D plots.
             The options for 'diagonal are:
                 - 'kde'
```

Comment:
1) Can we allow both strings and dicts for `kind` and have strings be broadcast across possible options for `lower`, `diagonal`, and `upper`? For example:
   * `'kde_1d'` --> `{'diagonal': 'kde_1d'}`
   * `'kde_2d'` --> `{'lower': 'kde_2d'}`
   * `'kde'` --> `{'diagonal': 'kde_1d', 'lower': 'kde_2d'}`
   * equivalently for `'hist'` instead of `'kde'`
2) `kind` or `kinds`? I'm ultimately happy with whichever feels to make more sense when you are trying to sort point (1) out.

Reply:
I agree. I've added in this string-based option in 0a3b7e0. For now I've gone with allowing both `'kind'` and `'kinds'` as synonyms.